lib/ansible/modules/cloud/amazon/aws_ssm_parameter_store.py

```diff
@@ -18,38 +21,46 @@
     description:
       - parameter key name.
     required: true
+    type: str
   description:
     description:
       - parameter key description.
     required: false
+    type: str
   value:
     description:
       - Parameter value.
     required: false
+    type: str
   state:
     description:
       - Creates or modifies an existing parameter
       - Deletes a parameter
     required: false
     choices: ['present', 'absent']
     default: present
+    type: str
   string_type:
     description:
       - Parameter String type
     required: false
     choices: ['String', 'StringList', 'SecureString']
     default: String
+    type: str
   decryption:
     description:
       - Work with SecureString type to get plain text secrets
     type: bool
     required: false
-    default: True
+    default: true
   key_id:
     description:
       - aws KMS key to decrypt the secrets.
+      - the default key (C(alias/aws/ssm)) is automatically generated the first
```

Comment:
```suggestion
      - The default key (C(alias/aws/ssm)) is automatically generated the first
```
and a period at the end of the sentence (in the next line). Maybe also clean up the sentence above :)